### ISSUE TITLE
Add readme note about packages to avoid in non-nixos

### ISSUE THREAD

**Issue by stites** [rycee/home-manager#593]
I've been getting a lot of segfaults when using home-manager on Ubuntu (a secondary box after a NixOS one) and have recently discovered that it's due to adding gcc7 (specifically `gcc7.out`) and maybe `coreutils` to my package list. In this scenario, I had both `~/.nix-profile/bin` taking precedence in my PATH, and `~/.nix-profile/lib` taking precedence in my LD_LIBRARY_PATH. My assumption is that somewhere there was a mix-up in compiling either Ubuntu's gcc was linking to nix's libraries or vice versa.

Long story short! I think that it might be worth putting a warning somewhere -- not sure where you think this would be appropriate.

**Comment by rycee** [comment]
Sorry about the little delay 🙂

Is this still an issue? If so then I think the [FAQ](https://github.com/rycee/home-manager/blob/master/FAQ.md) might be a good place. Something like "Why do I get segfaults on Ubuntu?"

**Comment by stites** [comment]
Thanks for following up! I ended up fixing the issue -- the problem was more of a nixpkgs-related problem than a home-manager-related one. I think a README warning would be nice, but I don't think this kind of warning should live in home-manager.

A home-manager solution to this problem could include verifying if you are working on a non-nixos box and, if so, warning a user if a nix-based gcc takes precedence in their PATH... but maybe this should only happen if more people sympathize with this scenario and file a separate issue with this proposal.

I don't think any further action needs to happen here!
